templates/nextjs/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "incremental": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
@emmatown this is a breaking change for people who have generated CJS projects before. Would it be possible to add a PSA somewhere?